flow360/component/simulation/translator/solver_translator.py

```diff
@@ -567,6 +578,87 @@ def translate_isosurface_output(
     return translated_output
 
 
+def translate_render_output(
+    input_params: SimulationParams,
+    output_params: list,
+    isosurface_injection_function,
+    slice_injection_function,
+):
+    """Translate render output settings."""
+
+    renders = get_all_entries_of_type(output_params, RenderOutput)
+
+    translated_outputs = []
+
+    for render in renders:
+        camera = render.camera.model_dump(exclude_none=True, exclude_unset=True, by_alias=True)
```

Comment:
```suggestion
render_dict = render.model_dump(exclude_none=True, exclude_unset=True, by_alias=True)
camera = render_dict["camera"]
```
This avoids duplicate model dump settings and are cleaner?